goentri.com.hotmart-subdomains.json

```diff
@@ -0,0 +1,28 @@
+{
+  "providerId": "goentri.com",
+  "providerName": "Entri",
+  "serviceId": "hotmart-subdomains",
+  "serviceName": "Hotmart Subdomains",
+  "version": 1,
+  "logoUrl": "https://cdn.goentri.com/logo.svg",
+  "description": "Allows user to easily set up dkim record using Entri",
+  "syncPubKeyDomain": "goentri.com",
+  "syncRedirectDomain": "api.goentri.com, goentri.com, entri.com, app.entri.com,app.goentri.com",
+  "variableDescription": "cnamePointsTo is the URL to where the subdomain is pointed and target is to where the root domains has to be pointed",
```

Comment:
sure about that the templates are not mixed up? This description tells about root domain and the service id is "subdomain"

Reply:
Maybe my description is not the best, the redirect is to redirect the root domain to the subdomain considering that the root domain is not pointed to anywhere in this case.

Reply:
But I deleted the Redirect anyway, they will not need that.